pkg/kubelet/kubelet.go

```diff
@@ -2055,14 +2069,32 @@ func (kl *Kubelet) isPodResourceUpdateAcceptable(pod *v1.Pod) bool {
 					return false
 				}
 			}
-			// TODO: Add ResizeResourcesStatus field to PodStatus, set to ResizeFailed, and update PodStatus
-			kl.recorder.Eventf(pod, v1.EventTypeWarning, events.FailedToResizePodResources, "Failed to update pod resources: %s", reason)
-			//TODO: restore the previous pod resources values
-			return false
+		} else {
+			// ResizeSuccessful
+			resizeStatus = v1.PodCondition{
+						Type:               v1.PodResourcesResizeStatus,
+						Status:             v1.ConditionTrue,
```

Comment:
is this used anywhere?
						Status:             v1.ConditionTrue,

Reply:
we don't use it at this time. but this can be used to report status, and may be used for rollback. We need to set some kind of status so that we complete the update cycle with the scheduler.